addon/templates/components/polaris-resource-list/item.hbs

```diff
@@ -1,140 +1,162 @@
-  id={{itemId}}
+  data-test-id="item-wrapper"
+  data-href={{@url}}
+  class="Polaris-ResourceList-Item
+    {{if this.focused "Polaris-ResourceList-Item--focused"}}
+    {{if this.selectable "Polaris-ResourceList-Item--selectable"}}
+    {{if this.selected "Polaris-ResourceList-Item--selected"}}
+    {{if this.selectMode "Polaris-ResourceList-Item--selectMode"}}
+    {{if @persistActions "Polaris-ResourceList-Item--persistActions"}}
+    {{if this.focusedInner "Polaris-ResourceList-Item--focusedInner"}}
+  "
+  {{on "click" this.handleClick}}
+  {{on "focusin" this.handleFocus}}
+  {{on "focusout" this.handleBlur}}
+  {{on "mousedown" this.handleMouseDown}}
+  {{on "keyup" this.handleKeypress}}
+  {{did-insert this.insertResourceListItem}}
+  ...attributes
 >
-  {{#if (or media selectable)}}
-    <div class="Polaris-ResourceList-Item__Owned">
-      {{#if selectable}}
-        {{#wrapper-element
-          tagName="div"
-          class="Polaris-ResourceList-Item__Handle"
-          data-test-id="larger-selection-area"
-          click=(action handleLargerSelectionArea)
-        }}
-          {{#wrapper-element
-            tagName="div"
-            class="Polaris-ResourceList-Item__CheckboxWrapper"
-            click=(action stopPropagation)
-          }}
-            {{polaris-checkbox
-              id=checkboxId
-              label=label
-              labelHidden=true
-              checked=isSelected
-              disabled=loading
-              onChange=(action handleSelection)
-            }}
-          {{/wrapper-element}}
-        {{/wrapper-element}}
-      {{/if}}
+  {{#let (if this.loading "-1" "0") as |tabIndex|}}
+    {{#if @url}}
+      <PolarisUnstyledLink
+        class="Polaris-ResourceList-Item__Link"
+        @ariaDescribedBy={{@itemId}}
+        @ariaLabel={{@accessibilityLabel}}
+        @url={{@url}}
+        @tabIndex={{tabIndex}}
+        @onFocus={{fn this.handleAnchorFocus}}
+        @onBlur={{fn this.handleFocusedBlur}}
+      />
+    {{else}}
+      {{#let (element "button") as |ItemButton|}}
+        <ItemButton
+          class="Polaris-ResourceList-Item__Button"
+          aria-label={{@accessibilityLabel}}
+          aria-controls={{@ariaControls}}
+          aria-expanded={{@ariaExpanded}}
+          tab-index={{tabIndex}}
+          {{on "click" this.handleClick}}
+          {{on "focusin" this.handleAnchorFocus}}
+          {{on "focusout" this.handleFocusedBlur}}
+        />
+      {{/let}}
+    {{/if}}
+  {{/let}}
 
-      {{#if media}}
-        <div class="Polaris-ResourceList-Item__Media" data-test-id="media">
-          {{render-content media}}
-        </div>
-      {{/if}}
-    </div>
-  {{/if}}
+  <div
+    data-test-id="item-content"
+    class="Polaris-ResourceList-Item__Container"
+    id={{@itemId}}
+  >
+    {{#if (or @media this.selectable)}}
+      <div class="Polaris-ResourceList-Item__Owned">
+        {{#if this.selectable}}
+          {{#let (element "div") as |ItemHandle|}}
```

Comment:
same....all these `let (element...` where there's no condition....we can just use the element tag directly no?